Assets/Scripts/Monster.cs

```diff
@@ -1,134 +1,42 @@
-using System;
-using System.Collections;
-using System.Collections.Generic;
 using System.Linq;
 using UnityEngine;
 
 
 [SelectionBase]
-public class Monster : MonoBehaviour
+public class Monster : Entity
 {
     [SerializeField]
     public Sprite deadSprite;
 
-    [SerializeField]
-    public float ouchForce = 19.5f;
-
-    [SerializeField]
-    public float startingHealth = 40;
-
-    [SerializeField] [InspectorReadonly]
-    private float _newtonHealth = 0;
-
-
     // component references
-    private new Rigidbody2D rigidbody;
-    private Accelerometer accelerometer;
+    /*private new Rigidbody2D rigidbody;*/
     private SpriteRenderer spriteRenderer;
     private ParticleSystem poofParticleSystem;
 
-    public bool isDead { get => _newtonHealth <= 0; }
-    private readonly float almostZero = 0.0001f;
-
     private void Awake()
     {
-        rigidbody = GetComponent<Rigidbody2D>();
-        accelerometer = GetComponent<Accelerometer>();
+        /*rigidbody = GetComponent<Rigidbody2D>();*/
         spriteRenderer = GetComponent<SpriteRenderer>();
         var particleSystems = GetComponentsInChildren<ParticleSystem>();
         poofParticleSystem = particleSystems.FirstOrDefault(p => p.gameObject.name == "PoofParticleSystem") ?? particleSystems[0];
     }
 
-    private void Start()
-    {
-        _newtonHealth = startingHealth;
-    }
-
-    private void FixedUpdate()
+    private new void OnCollisionEnter2D(Collision2D collision)
     {
-        
-    }
-
-    private void OnCollisionEnter2D(Collision2D collision)
-    {
-        CollisionHandler(collision);
-    }
-
-    private void CollisionHandler(Collision2D collision)
-    {
-        if (isDead) return;
-
-        // if collision is with a bird, always consider it a kill
-        Bird bird = collision.gameObject.GetComponent<Bird>();
-        if (bird)
+        if (isDead)
```

Comment:
I think this belongs in the base class all together.

Reply:
Done: https://github.com/cjh15hub/terribletweeters/pull/1/commits/bf5906843a21d2e978aa0e82765b2f23f2a3315e#diff-8446b20d766823004b26f3f1b33f5831cf745c6d4a21828ccea0479ff1c54ce9R46